packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationModule.kt

```diff
@@ -47,23 +36,23 @@ class LocalizationModule(context: Context) : ExportedModule(context) {
       val isRTL = TextUtils.getLayoutDirectionFromLocale(locale) == View.LAYOUT_DIRECTION_RTL
       val region = getRegionCode(locale)
       val symbols = DecimalFormatSymbols(locale)
-      return Bundle().apply {
-        putString("currency", getCurrencyCode(locale))
-        putString("decimalSeparator", symbols.decimalSeparator.toString())
-        putString("digitGroupingSeparator", symbols.groupingSeparator.toString())
-        putStringArrayList("isoCurrencyCodes", iSOCurrencyCodes)
-        putBoolean("isMetric", !USES_IMPERIAL.contains(region))
-        putBoolean("isRTL", isRTL)
-        putString("locale", localeNames[0])
-        putStringArrayList("locales", localeNames)
-        putString("region", region)
-        putString("timezone", TimeZone.getDefault().id)
-      }
+      return bundleOf(
+        "currency" to getCurrencyCode(locale),
+        "decimalSeparator" to symbols.decimalSeparator.toString(),
+        "digitGroupingSeparator" to symbols.groupingSeparator.toString(),
+        "isoCurrencyCodes" to ISOCurrencyCodes,
+        "isMetric" to !USES_IMPERIAL.contains(region),
+        "isRTL" to isRTL,
+        "locale" to localeNames[0],
+        "locales" to localeNames,
+        "region" to region,
+        "timezone" to TimeZone.getDefault().id
+      )
     }
 
   private val locales: ArrayList<Locale>
     get() {
-      val context = applicationContext ?: return ArrayList()
+      val context = appContext.reactContext ?: return ArrayList()
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/empty-list.html
```suggestion
      val context = appContext.reactContext ?: emptyList()
```